src/multi_vector_simulator/D0_modelling_and_optimization.py

```diff
@@ -101,8 +101,7 @@ def run_oemof(dict_values, save_energy_system_graph=False, save_lp_file=False):
     local_energy_system = D2.add_constraints(
         local_energy_system, dict_values, dict_model
     )
-    if save_lp_file is True:
-        model_building.store_lp_file(dict_values, local_energy_system)
+    model_building.store_lp_file(dict_values, local_energy_system)
```

Comment:
@Bachibouzouk, in the changelog you wrote 
"Write lp file only when executing `cli.py` (#675)"
, but here it looks like it will be stored for every simulation. Could you clarify this please?

Reply:
I forgot to add "if user set the `OUTPUT_LP_FILE` setting to `True`"